docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,67 @@
+# Capture
+
+
+Capture snapshot the current DALi scene. The captured result can be saved as a file and also can be used to be rendered by DALi `ImageView` or `ImageVisual`.
+
+In this tutorial, we show how to capture scene and to use it.
+
+
+## Tutorial
+
+Applications should follow the example below to use `Dali::Capture`.
+
+``` C++
+Capture capture = Capture::New();
+
+capture.Start( source, size, path, clearColor, quality);
+```
+
+Each parameter of `Capture::Start` is denoted as follows:
+
+ - **Actor source**: source actor to be rendered scene. Only this source actor and its children are captured.
+ - **Vector2 size**: size of captured result image.
+ - **std::string path**: system path to be saved as a file. If the path is empty, `Dali::Capture` do not save the image as file and the captured result can be used to be rendered by DALi `ImageView` or `ImageVisual`.
+ - **Vector4 clearcolor**: background color of captured scene
+ - **int quality**: a value to control quality of saved image file for jpeg file format in the range [1, 100]
```

Comment:
question: does it save only in JPEG format, what about range for other formats?
Please rephrase for clarity, see below suggestion, please verify?

a value to control the quality of the saved image file. For JPEG file format, the value must be in the range [1,100].

Reply:
Thank you. And I added new information of file format that can be used to saved file.